x-pack/plugins/security_solution/public/common/components/exceptions/translations.ts

```diff
@@ -47,3 +47,60 @@ export const OPERATING_SYSTEM = i18n.translate(
     defaultMessage: 'OS',
   }
 );
+
+export const SEARCH_DEFAULT = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.searchDefaultPlaceholder',
+  {
+    defaultMessage: 'Search field (default)',
+  }
+);
+
+export const ADD_EXCEPTION_LABEL = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.addExceptionLabel',
+  {
+    defaultMessage: 'Add new exception',
+  }
+);
+
+export const ADD_TO_ENDPOINT_LIST = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.addToEndpointListLabel',
+  {
+    defaultMessage: 'Add to endpoint list',
+  }
+);
+
+export const ADD_TO_DETECTIONS_LIST = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.addToDetectionsListLabel',
+  {
+    defaultMessage: 'Add to detections list',
+  }
+);
+
+export const EXCEPTION_EMPTY_PROMPT_TITLE = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.emptyPromptTitle',
+  {
+    defaultMessage: 'You have no exceptions',
+  }
+);
+
+export const EXCEPTION_EMPTY_PROMPT_BODY = i18n.translate(
+  'xpack.securitySolution.exceptions.viewer.emptyPromptBody',
+  {
+    defaultMessage:
+      'This could be a description of what exceptions are and what their use case could be. Wow such tempt much ruin diet wrinkler clouds, you are doin me a concern. H*cksub woofer shibe clouds.',
```

Comment:
@marrasherrier @dontcallmesherryli Could you please help us with the copy for this empty state?

![image](https://user-images.githubusercontent.com/616158/83817266-487fea00-a692-11ea-81d2-358cc075f2d7.png)

Reply:
You can add an exception to fine tune the rule so that it ignores designated ECS fields. Exceptions leverage detection accuracy, which can help reduce the number of false positives.

Reply:
Updated!